TaniaGayosso/HackerRank_August25/IfElse.java

```diff
@@ -0,0 +1,29 @@
+package ChallengesOne;
```

Comment:
Is not usual name a package with upper case letters. We can use Google Java Style Guide:
https://google.github.io/styleguide/javaguide.html#s5-naming
Please check 5.2.1